imqsauth/http.go

```diff
@@ -1079,6 +1080,36 @@ func httpHandlerRenameUser(central *ImqsCentral, w http.ResponseWriter, r *httpR
 	authaus.HttpSendTxt(w, http.StatusOK, "Renamed '"+oldIdent+"' to '"+newIdent+"'")
 }
 
+func httpHandlerGetGroupsPermNames(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
```

Comment:
Can you please add a comment above this function describing what it returns and why it exists?

Reply:
Done